src/devices/Arduino/Arduino.sln

```diff
@@ -0,0 +1,93 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29905.134
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Arduino", "Arduino.csproj", "{0D3AA0FD-4834-4AB1-AD2A-BB12180B61B4}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Device.Gpio", "..\..\System.Device.Gpio\System.Device.Gpio.csproj", "{0B90F9D4-7353-4172-A317-714471A06781}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Arduino.sample", "samples\Arduino.sample.csproj", "{2670F7BF-A7C8-49EB-9A99-1719A90D0C67}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Bmxx80", "..\Bmxx80\Bmxx80.csproj", "{EEEB0FB8-E1ED-4970-BDF6-DA3D5E2ED074}"
```

Comment:
NIT: Do you need to include these projects? if they are just project references, you don't need to have them in the solution file.

Reply:
Intellisense and Resharper are acting weird if the projects are not part of the solution. They fail to recognize the types from the referenced libraries if they're not part of the solution.

Reply:
Yeah I’ve seen that happen too. The way to fix that is to open the settings in VS and disable nuget package restore of missing packages from the nuget options. Basically jn order for VS to understand how to use restore correctly it requires all referenced projects to be part of the solution, but if you disable restore from VS and just run restore from the command line before opening VS then VS will work as expected and won’t require the projects to be part of the solution.